src/common/wheeled_base.rs

```diff
@@ -14,18 +13,18 @@ use std::sync::{Arc, Mutex};
 pub(crate) fn register_models(registry: &mut ComponentRegistry) {
     if registry
         .register_base(
-            "esp32_wheeled_base",
-            &Esp32WheelBase::<MotorType, MotorType>::from_config,
+            "micro_rdk_wheeled_base",
```

Comment:
Why put the prefix in at all, rather than just "wheeled_base"? Or even just `wheeled` to match RDK?

Reply:
you're right, I'm actually going to name it "two-wheeled base". Originally I was trying to potentially accommodate https://viam.atlassian.net/browse/RSDK-5412. But since we're hardcoding the models for that ticket it doesn't matter